doc/arch/adr-002-handle-race-conditions-in-govuk-index.md

```diff
@@ -0,0 +1,105 @@
+# Decision record: use versioning to handle race conditions when updating the govuk index
```

Comment:
@suzannehamilton I think this is a better title. 

@MatMoore wanted to clarify your understanding of the best way to write the title for ADR's.

I like what @suzannehamilton has here but would have leaned towards "Handling Race conditions when updating govuk index" as that describes the problem we are making a decision about while adding "use versioning" describes the solution as well. 

Which way should we lean? Are both valid depending on personal preference?